lms/djangoapps/completion/tests/test_models.py

```diff
@@ -142,3 +144,73 @@ def test_cannot_call_submit_completion(self):
                 completion=0.9,
             )
         self.assertEqual(models.BlockCompletion.objects.count(), 1)
+
+
+@ddt.ddt
+class AggregateCompletionTestCase(TestCase, CompletionSetUpMixin):
+
+    @classmethod
+    def setUpClass(cls):
+        super(AggregateCompletionTestCase, cls).setUpClass()
+        cls.overrider = waffle.waffle().override(waffle.ENABLE_COMPLETION_TRACKING, True)
+        cls.overrider.__enter__()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.overrider.__exit__(None, None, None)
+        super(AggregateCompletionTestCase, cls).tearDownClass()
+
+    def setUp(self):
+        super(AggregateCompletionTestCase, self).setUp()
+        self.set_up_completion()
+
+    def test_submit_completion_with_valid_data(self):
+        aggregate_name = 'course'
+        earned = 0.5
+        possible = 1
+        obj, isNew = models.AggregateCompletion.objects.submit_completion(self.user, self.block_key.course_key, self.block_key, aggregate_name, earned, possible)
+        self.assertTrue(isNew)
+        self.assertEqual(len(models.AggregateCompletion.objects.all()), 1)
+        model = models.AggregateCompletion.objects.first()
+        self.assertEqual(model.id, obj.id)
+
+    @ddt.data(
+        # Earned greater than possible
+        (u'block-v1:edx+test+run+type@video+block@doggos', 'course', 1.1, 1),
```

Comment:
These are strings, not `CourseKey` objects, and so they all fail for the same reason, not the different cases you're trying to test.